ads/inabox/inabox-messaging-host.js

```diff
@@ -122,15 +122,24 @@ export class InaboxMessagingHost {
       return false;
     }
 
-    const iframe =
+    const adFrame =
         this.getFrameElement_(message.source, request['sentinel']);
-    if (!iframe) {
+    if (!adFrame) {
       dev().info(TAG, 'Ignored message from untrusted iframe:', message);
       return false;
     }
 
+    const allowedTypes = adFrame.iframe.dataset['ampAllowed'];
+    const hasAllowedTypeRegex =
+        new RegExp(`(^|,)\\s*${request['type']}\\s*($|,)`);
+    // having no whitelist is legacy behavior so assume all types are allowed
```

Comment:
I actually spoke with Jeff about this and I believe we agreed that in absence of a whitelist we should assume read-only operations are allowed but not write (e.g. resize).

Reply:
We need to not break creatives that are using this properly, and while I don't currently think there are any there are two ways to find out:

* The current plan: no whitelist means allow everything, ad tags start putting a limited whitelist on all iframes, if that experiment looks good then we know we haven't broken anything

* Alternate plan: add beacons for when we would reject something but don't reject it yet.  If we don't get these beacons then we know we haven't broken anything.

In both cases, after we're sure that this didn't break anything we switch to "absence of a whitelist we should assume read-only operations are allowed but not write"